Scripts_BeyzanurÖzen/Player.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player : MonoBehaviour
+{
+
+	void OnTriggerStay(Collider other)
+	{
+		var Usable = other.gameObject.GetComponent<IUsable>();
+		if (Usable != null)
+		{
+			Usable.Text();
+		}
+		Usable.Action();

```

Comment:
Bu da if check in icinde olmali

Reply:
Player Update loop'unda Usable var mi ve E tusuna basildimi diye bakip Action i cagirmalisin